docs/src/man/hosting.md

```diff
@@ -428,7 +435,7 @@ jobs:
               git config user.email "documenter@juliadocs.github.io"
               git rm -rf "${preview_dir}"
               git commit -m "delete preview"
-              git branch gh-pages-new $(echo "delete history" | git commit-tree HEAD^{tree})
+              git branch gh-pages-new "$(echo "delete history" | git commit-tree "HEAD^{tree}")"
```

Comment:
Quoting here avoids issues with this shellcheck warning: https://www.shellcheck.net/wiki/SC2086